packages/styles/scss/components/filter-panel/_filter-panel.scss

```diff
@@ -0,0 +1,426 @@
+      }
+      &:hover {
+        color: $hover-primary-text;
+      }
+    }
+
+    &:not([has-selections]) {
+      .#{$prefix}--clear {
+        display: none;
+      }
+    }
+    @include carbon--breakpoint-down('lg') {
+      display: none;
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-panel-composite) {
+    .#{$prefix}--filter-button {
+      @include use-carbon-productive-tokens();
+      @include carbon--type-style('body-short-01');
+
+      cursor: pointer;
+      border: 1px solid $interactive-03;
+      color: $interactive-03;
+      padding: $spacing-05;
+      margin-top: $spacing-05;
+      margin-left: $spacing-05;
+      background-color: transparent;
+      width: calc((#{$spacing-10} + #{($spacing-10)}));
+      @include carbon--breakpoint('lg') {
+        display: none;
+      }
+      &:hover {
+        background-color: $hover-tertiary;
+        color: $inverse-01;
+      }
+    }
+  }
+
+  :host(#{$dds-prefix}-checkbox) {
+    padding-left: calc((#{$spacing-04} + #{($spacing-01)}));
+    padding-bottom: calc((#{$spacing-01} + #{($spacing-02)}));
+    @extend :host(#{$prefix}-checkbox);
+    @include use-carbon-productive-tokens();
+    @include carbon--type-style('body-short-01');
+
+    color: $text-02;
+    &:hover {
+      color: $text-01;
+    }
+    &[checked] {
+      color: $text-01;
+    }
+    label.#{$prefix}--checkbox-label {
+      padding-top: $spacing-01;
+      padding-left: $spacing-06;
+    }
+    label.#{$prefix}--checkbox-label::before {
+      border: 1px solid $icon-02;
+    }
+    label.#{$prefix}--checkbox-label:hover::before {
+      border: 1px solid $icon-01;
+    }
+  }
+  :host(#{$dds-prefix}-input-select) {
+    &:hover {
+      background-color: $hover-ui;
+    }
+    .#{$prefix}--input_container {
+      &:focus-visible {
+        outline: $spacing-01 solid $focus;
+      }
+    }
+
+    display: block;
+    &[selected] {
+      background-color: $highlight;
+      .#{$prefix}--input_container-heading {
+        color: $text-01;
+        &:focus-visible {
+          outline: $spacing-01 solid $focus;
+        }
+      }
+    }
+    .#{$prefix}--input_container-heading {
+      &:focus-visible {
+        outline: $spacing-01 solid $focus;
+        margin-right: $spacing-01;
+        margin-left: $spacing-01;
+        padding-left: calc((#{$spacing-04} + #{($spacing-01)}));
+      }
+    }
+  }
+  :host(#{$dds-prefix}-input-select-item) {
+    @include use-carbon-productive-tokens();
+    @include carbon--type-style('body-short-01');
+
+    background-color: $ui-01;
+    width: 100%;
+    cursor: pointer;
+    position: relative;
+    color: $text-02;
+    padding: $spacing-03 $spacing-06 $spacing-03 $spacing-06;
+
+    &:hover {
+      background-color: $hover-ui;
+      color: $text-01;
+    }
+    &:active {
+      background-color: $highlight;
+    }
+
+    &[selected] {
+      background-color: $highlight;
+      color: $text-01;
+    }
+    .#{$prefix}--close_icon {
```

Comment:
This looks like a duplicate. Seems like most of the styles for `bx--input_container-heading` and `dds-input-select-item` can be combined?